daily-planner/lib/ai/llm/outputParser.ts

```
@@ -0,0 +1,77 @@
+      isStrArr(obj.themes) &&
+      isStrArr(obj.action_items) &&
+      typeof obj.tone === 'string'
+    );
+  }
+
+  /** Remove problematic content; conservative sanitization */
+  static sanitizeContent<T extends StructuredSummary>(obj: T): T {
+    const scrub = (s: string) =>
+      s
+        .replace(/[\u200B-\u200D\uFEFF]/g, '') // zero-width
+        .replace(/[\x00-\x08\x0B-\x0C\x0E-\x1F]/g, '') // ctrl
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)